scripts/update_from_template.sh

```diff
@@ -8,4 +8,5 @@ git checkout -B feature/update_from_template; git pull
 git merge template/main --allow-unrelated-histories -m 'feat: pull changes from remote template'
 git checkout --ours pixi.lock; git add pixi.lock
 git remote remove template
-git push --set-upstream origin feature/update_from_template
\ No newline at end of file
+git push --set-upstream origin feature/update_from_template
+git checkout main
```

Comment:
What is the reasoning behind checking out main?

I think I left on the update_from_template branch because I often make other minor adjustments if there are any subtleties related to any merge conflicts.

Reply:
This is not really an important requirement. Only that this branch is being created opaquely and the user may continue making changes in this branch not realising they are now working in a new branch, and push main to remote, where no changes were made.

P.S. I did have merge conflict issues as well a couple of times, and had to resolve myself. But in this case the checkout to main will fail anyway.